config/vxlan.py

```diff
@@ -24,6 +25,8 @@ def add_vxlan(db, vxlan_name, src_ip):
     if ADHOC_VALIDATION:
         if not clicommon.is_ipaddress(src_ip):
             ctx.fail("{} invalid src ip address".format(src_ip))  
+        if len(vxlan_name) >= IFNAMSIZ:
+           ctx.fail("'vxlan_name' length should not exceed {} characters".format(IFNAMSIZ))
```

Comment:
@arfeigin why not to reuse name validator?
```python
from utilities_common.helper import validate_interface_name_length
```

Reply:
fixed